packages/web/app/src/components/ui/empty-list.tsx

```diff
@@ -22,7 +22,10 @@ export const EmptyList = ({
 }): ReactElement => {
   return (
     <Card
-      className={cn('flex grow cursor-default flex-col items-center gap-y-2 p-4', className)}
+      className={cn(
+        'flex max-h-screen min-h-[400px] grow cursor-default flex-col items-center gap-y-2 p-4',
```

Comment:
Addresses when the list of checks is very long. It causes this element to grow to where the text is all off screen.